src/components/ScrollToTop.astro

```diff
@@ -0,0 +1,78 @@
+---
+const { default: arrowUp } = await import(
+  "../../public/img/icons/arrow-up.svg?raw"
+);
+---
+
+<div id="scroll-to-top">
```

Comment:
since this is clickable, it should be a `<button>` as a semantic. Also, onClick can be added directly in HTML, calling the global function `scrollToTop`.

Reply:
Replaced the `div` with the `button` tag, but I was not able to add the eventListener on `onclick`